### PR TITLE
Change legalities of historic and standard brawl

### PR DESCRIPTION
**Pull request by Bassiuz**
Scryfall changed the way legalities were tracked regarding to brawl. They seem to have stopped tracking historic brawl (treating that as normal brawl) and started tracking standardbrawl now.

Without this change, getting cards will break the Json Serialization check and the package breaks.

### CODE REVIEW

File: test/models/mtg_card_test.dart [hunk 1]
```diff
@@ -12,10 +12,8 @@ void main() {
         final name = 'Huang Zhong, Shu General';
         final langStr = 'en';
         final releasedAtStr = '2016-11-16';
-        final uriStr =
-            'https://api.scryfall.com/cards/1ebf9dea-b4f8-4955-9824-7da5bbba91e9';
-        final scryfallUriStr =
-            'https://scryfall.com/card/pz2/65801/huang-zhong-shu-general?utm_source=api';
+        final uriStr = 'https://api.scryfall.com/cards/1ebf9dea-b4f8-4955-9824-7da5bbba91e9';
+        final scryfallUriStr = 'https://scryfall.com/card/pz2/65801/huang-zhong-shu-general?utm_source=api';
```
Comment (aroningruber) [inline test/models/mtg_card_test.dart:16]: Please revert unrelated formatting changes.

File: test/models/mtg_card_test.dart [hunk 2]
```diff
@@ -36,8 +34,7 @@ void main() {
         final manaCost = '{2}{W}{W}';
         final cmc = 4;
         final typeLine = 'Legendary Creature — Human Soldier';
-        final oracleText =
-            'Huang Zhong, Shu General can\'t be blocked by more than one creature.';
+        final oracleText = 'Huang Zhong, Shu General can\'t be blocked by more than one creature.';
```
Comment (aroningruber) [inline test/models/mtg_card_test.dart:37]: Please revert unrelated formatting changes.

File: test/models/mtg_card_test.dart [hunk 3]
```diff
@@ -77,21 +74,16 @@ void main() {
...
-        final setSearchUriStr =
-            'https://api.scryfall.com/cards/search?order=set&q=e%3Apz2&unique=prints';
-        final scryfallSetUriStr =
-            'https://scryfall.com/sets/pz2?utm_source=api';
-        final rulingsUriStr =
-            'https://api.scryfall.com/cards/1ebf9dea-b4f8-4955-9824-7da5bbba91e9/rulings';
+        final setUriStr = 'https://api.scryfall.com/sets/2661b143-8eac-4c73-9d93-549fe928bd96';
+        final setSearchUriStr = 'https://api.scryfall.com/cards/search?order=set&q=e%3Apz2&unique=prints';
+        final scryfallSetUriStr = 'https://scryfall.com/sets/pz2?utm_source=api';
+        final rulingsUriStr = 'https://api.scryfall.com/cards/1ebf9dea-b4f8-4955-9824-7da5bbba91e9/rulings';
```
Comment (aroningruber) [inline test/models/mtg_card_test.dart:80]: Please revert unrelated formatting changes.

File: test/models/mtg_card_test.dart [hunk 4]
```diff
@@ -77,21 +74,16 @@ void main() {
...
         final printsSearchUriStr =
             'https://api.scryfall.com/cards/search?order=released&q=oracleid%3A7d6c4290-d46d-4b98-805c-3f537462c4c8&unique=prints';
         final collectorNumber = '65801';
         final digital = true;
         final rarityStr = 'rare';
-        final flavorText =
-            '\'Virile in war, he kept the north in fear; His prodigies subdued the western sphere.\'';
+        final flavorText = '\'Virile in war, he kept the north in fear; His prodigies subdued the western sphere.\'';
```
Comment (aroningruber) [inline test/models/mtg_card_test.dart:86]: Please revert unrelated formatting changes.

File: test/models/mtg_card_test.dart [hunk 5]
```diff
@@ -118,8 +110,7 @@ void main() {
           'tcgplayer_infinite_decks':
               'https://infinite.tcgplayer.com/search?contentMode=deck&game=magic&partner=scryfall&q=Huang+Zhong%2C+Shu+General&utm_campaign=affiliate&utm_medium=api&utm_source=scryfall',
           'edhrec': 'https://edhrec.com/route/?cc=Huang+Zhong%2C+Shu+General',
-          'mtgtop8':
-              'https://mtgtop8.com/search?MD_check=1&SB_check=1&cards=Huang+Zhong%2C+Shu+General'
+          'mtgtop8': 'https://mtgtop8.com/search?MD_check=1&SB_check=1&cards=Huang+Zhong%2C+Shu+General'
```
Comment (aroningruber) [inline test/models/mtg_card_test.dart:113]: Please revert unrelated formatting changes.

File: test/models/mtg_card_test.dart [hunk 6]
```diff
@@ -131,8 +122,7 @@ void main() {
         };
         final preview = {
           'source': 'Sjow',
-          'source_uri':
-              'https://twitter.com/LiquidSjow/status/1211727247247388672?s=19',
+          'source_uri': 'https://twitter.com/LiquidSjow/status/1211727247247388672?s=19',
```
Comment (aroningruber) [inline test/models/mtg_card_test.dart:125]: Please revert unrelated formatting changes.

File: test/models/mtg_card_test.dart [hunk 7]
```diff
@@ -142,17 +132,15 @@ void main() {
             'component': 'combo_piece',
             'name': 'Ruthless Knave',
             'type_line': 'Creature — Orc Pirate',
-            'uri':
-                'https://api.scryfall.com/cards/1ce91e38-4601-4354-ad1b-2c5c1c70da17'
+            'uri': 'https://api.scryfall.com/cards/1ce91e38-4601-4354-ad1b-2c5c1c70da17'
```
Comment (aroningruber) [inline test/models/mtg_card_test.dart:135]: Please revert unrelated formatting changes.

File: test/models/mtg_card_test.dart [hunk 8]
```diff
@@ -142,17 +132,15 @@ void main() {
...
           },
           {
             'object': 'related_card',
             'id': '720f3e68-84c0-462e-a0d1-90236ccc494a',
             'component': 'token',
             'name': 'Treasure',
             'type_line': 'Token Artifact — Treasure',
-            'uri':
-                'https://api.scryfall.com/cards/720f3e68-84c0-462e-a0d1-90236ccc494a'
+            'uri': 'https://api.scryfall.com/cards/720f3e68-84c0-462e-a0d1-90236ccc494a'
```
Comment (aroningruber) [inline test/models/mtg_card_test.dart:143]: Please revert unrelated formatting changes.

File: test/models/paginable_list_test.dart [hunk 9]
```diff
@@ -118,13 +112,10 @@ void main() {
...
-            'set_uri':
-                'https://api.scryfall.com/sets/985eab7d-655a-4cb0-ba74-d48c8dcfb3d4',
-            'set_search_uri':
-                'https://api.scryfall.com/cards/search?order=set&q=e%3Achr&unique=prints',
+            'set_uri': 'https://api.scryfall.com/sets/985eab7d-655a-4cb0-ba74-d48c8dcfb3d4',
+            'set_search_uri': 'https://api.scryfall.com/cards/search?order=set&q=e%3Achr&unique=prints',
             'scryfall_set_uri': 'https://scryfall.com/sets/chr?utm_source=api',
-            'rulings_uri':
-                'https://api.scryfall.com/cards/023b5e6f-10de-422d-8431-11f1fdeca246/rulings',
+            'rulings_uri': 'https://api.scryfall.com/cards/023b5e6f-10de-422d-8431-11f1fdeca246/rulings',
```
Comment (aroningruber) [inline test/models/paginable_list_test.dart:118]: Please revert unrelated formatting changes.

File: lib/src/models/legalities.dart [hunk 10]
```diff
@@ -102,6 +102,5 @@ class Legalities {
   });
 
   /// Constructs a [Legalities] object from JSON.
-  factory Legalities.fromJson(Map<String, dynamic> json) =>
-      _$LegalitiesFromJson(json);
+  factory Legalities.fromJson(Map<String, dynamic> json) => _$LegalitiesFromJson(json);
```
Comment (aroningruber) [inline lib/src/models/legalities.dart:105]: Please revert unrelated formatting changes.

File: test/models/paginable_list_test.dart [hunk 11]
```diff
@@ -16,9 +16,7 @@ void main() {
 
         expect(
           PaginableList<int>.fromJson(json, (o) => o as int),
-          isA<PaginableList<int>>()
-              .having((l) => l.hasMore, 'hasMore', hasMore)
-              .having((l) => l.data, 'data', data),
+          isA<PaginableList<int>>().having((l) => l.hasMore, 'hasMore', hasMore).having((l) => l.data, 'data', data),
```
Comment (aroningruber) [inline test/models/paginable_list_test.dart:19]: Please revert unrelated formatting changes.

File: test/models/paginable_list_test.dart [hunk 12]
```diff
@@ -33,9 +31,7 @@ void main() {
 
         expect(
           PaginableList<String>.fromJson(json, (o) => o as String),
-          isA<PaginableList<String>>()
-              .having((l) => l.hasMore, 'hasMore', hasMore)
-              .having((l) => l.data, 'data', data),
+          isA<PaginableList<String>>().having((l) => l.hasMore, 'hasMore', hasMore).having((l) => l.data, 'data', data),
```
Comment (aroningruber) [inline test/models/paginable_list_test.dart:34]: Please revert unrelated formatting changes.

File: test/models/paginable_list_test.dart [hunk 13]
```diff
@@ -54,10 +50,8 @@ void main() {
             'name': 'Abu Ja\'far',
             'lang': 'en',
             'released_at': '1995-07-01',
-            'uri':
-                'https://api.scryfall.com/cards/023b5e6f-10de-422d-8431-11f1fdeca246',
-            'scryfall_uri':
-                'https://scryfall.com/card/chr/1/abu-jafar?utm_source=api',
+            'uri': 'https://api.scryfall.com/cards/023b5e6f-10de-422d-8431-11f1fdeca246',
+            'scryfall_uri': 'https://scryfall.com/card/chr/1/abu-jafar?utm_source=api',
```
Comment (aroningruber) [inline test/models/paginable_list_test.dart:54]: Please revert unrelated formatting changes.

File: test/models/paginable_list_test.dart [hunk 14]
```diff
@@ -150,15 +141,13 @@ void main() {
...
-                  'https://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=2838',
+              'gatherer': 'https://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=2838',
               'tcgplayer_infinite_articles':
                   'https://infinite.tcgplayer.com/search?contentMode=article&game=magic&partner=scryfall&q=Abu+Ja%27far&utm_campaign=affiliate&utm_medium=api&utm_source=scryfall',
               'tcgplayer_infinite_decks':
                   'https://infinite.tcgplayer.com/search?contentMode=deck&game=magic&partner=scryfall&q=Abu+Ja%27far&utm_campaign=affiliate&utm_medium=api&utm_source=scryfall',
               'edhrec': 'https://edhrec.com/route/?cc=Abu+Ja%27far',
-              'mtgtop8':
-                  'https://mtgtop8.com/search?MD_check=1&SB_check=1&cards=Abu+Ja%27far'
+              'mtgtop8': 'https://mtgtop8.com/search?MD_check=1&SB_check=1&cards=Abu+Ja%27far'
```
Comment (aroningruber) [inline test/models/paginable_list_test.dart:150]: Please revert unrelated formatting changes.

File: test/models/paginable_list_test.dart [hunk 15]
```diff
@@ -179,10 +168,8 @@ void main() {
             'name': 'Adorable Kitten',
             'lang': 'en',
             'released_at': '2020-02-29',
-            'uri':
-                'https://api.scryfall.com/cards/d25ff6aa-a01d-49f2-926f-8f5457143b5c',
-            'scryfall_uri':
-                'https://scryfall.com/card/und/1/adorable-kitten?utm_source=api',
+            'uri': 'https://api.scryfall.com/cards/d25ff6aa-a01d-49f2-926f-8f5457143b5c',
+            'scryfall_uri': 'https://scryfall.com/card/und/1/adorable-kitten?utm_source=api',
```
Comment (aroningruber) [inline test/models/paginable_list_test.dart:172]: Please revert unrelated formatting changes.

File: test/models/paginable_list_test.dart [hunk 16]
```diff
@@ -243,13 +230,10 @@ void main() {
...
-            'set_uri':
-                'https://api.scryfall.com/sets/fccfdf97-f5f2-43b4-9be9-9255414e6633',
-            'set_search_uri':
-                'https://api.scryfall.com/cards/search?order=set&q=e%3Aund&unique=prints',
+            'set_uri': 'https://api.scryfall.com/sets/fccfdf97-f5f2-43b4-9be9-9255414e6633',
+            'set_search_uri': 'https://api.scryfall.com/cards/search?order=set&q=e%3Aund&unique=prints',
             'scryfall_set_uri': 'https://scryfall.com/sets/und?utm_source=api',
-            'rulings_uri':
-                'https://api.scryfall.com/cards/d25ff6aa-a01d-49f2-926f-8f5457143b5c/rulings',
+            'rulings_uri': 'https://api.scryfall.com/cards/d25ff6aa-a01d-49f2-926f-8f5457143b5c/rulings',
```
Comment (aroningruber) [inline test/models/paginable_list_test.dart:236]: Please revert unrelated formatting changes.

File: test/models/paginable_list_test.dart [hunk 17]
```diff
@@ -274,15 +258,13 @@ void main() {
...
-                  'https://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=479485',
+              'gatherer': 'https://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=479485',
               'tcgplayer_infinite_articles':
                   'https://infinite.tcgplayer.com/search?contentMode=article&game=magic&partner=scryfall&q=Adorable+Kitten&utm_campaign=affiliate&utm_medium=api&utm_source=scryfall',
               'tcgplayer_infinite_decks':
                   'https://infinite.tcgplayer.com/search?contentMode=deck&game=magic&partner=scryfall&q=Adorable+Kitten&utm_campaign=affiliate&utm_medium=api&utm_source=scryfall',
               'edhrec': 'https://edhrec.com/route/?cc=Adorable+Kitten',
-              'mtgtop8':
-                  'https://mtgtop8.com/search?MD_check=1&SB_check=1&cards=Adorable+Kitten'
+              'mtgtop8': 'https://mtgtop8.com/search?MD_check=1&SB_check=1&cards=Adorable+Kitten'
```
Comment (aroningruber) [inline test/models/paginable_list_test.dart:267]: Please revert unrelated formatting changes.

File: test/models/paginable_list_test.dart [hunk 18]
```diff
@@ -323,58 +305,49 @@ void main() {
...
+            'search_uri': 'https://api.scryfall.com/cards/search?order=set&q=e%3Apw22&unique=prints',
             'released_at': '2022-03-05',
             'set_type': 'promo',
             'card_count': 3,
             'digital': false,
             'nonfoil_only': false,
             'foil_only': true,
-            'icon_svg_uri':
-                'https://c2.scryfall.com/file/scryfall-symbols/sets/star.svg?1640581200'
+            'icon_svg_uri': 'https://c2.scryfall.com/file/scryfall-symbols/sets/star.svg?1640581200'
```
Comment (aroningruber) [inline test/models/paginable_list_test.dart:350]: Please revert unrelated formatting changes.

File: test/models/paginable_list_test.dart [hunk 19]
```diff
@@ -389,9 +362,7 @@ void main() {
             json,
             (o) => MtgSet.fromJson(o as Map<String, dynamic>),
           ),
-          isA<PaginableList<MtgSet>>()
-              .having((l) => l.hasMore, 'hasMore', hasMore)
-              .having(
+          isA<PaginableList<MtgSet>>().having((l) => l.hasMore, 'hasMore', hasMore).having(
```
Comment (aroningruber) [inline test/models/paginable_list_test.dart:365]: Please revert unrelated formatting changes.

**Review by aroningruber — CHANGES_REQUESTED** [review]
Thank you very much for your PR.

Please revert any unrelated formatting changes to keep this PR focused.

**Comment by Bassiuz** [comment]
@aroningruber You're completely right, my linter was off. Reverted those changes and fixed it without formatting

**Review by aroningruber — APPROVED** [review]
